.env.template

```diff
@@ -0,0 +1,4 @@
+# Don't commit the .env file to version control!
```

Comment:
Welp.

Reply:
Wau?

Reply:
Oh it's a template you should copy over into your own project. :crab: